contracts/AggregatorV2V3Interface.sol

```diff
@@ -0,0 +1,56 @@
+pragma solidity >=0.5.0;
+
+import "./AggregatorInterface.sol";
```

Comment:
These imports are unused. Looks like we can't inherit interfaces in 0.5?

If that's the case, we can probably remove `AggregatorInterface.sol` and `AggregatorV3Interface.sol` altogether and just use this contract.